.goreleaser.yaml

```diff
@@ -0,0 +1,35 @@
+version: 1
+
+project_name: ui-plugin-operator
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -w -X "github.com/rancher/ui-plugin-operator/pkg/version.Version=v{{ .Version }}" -X "github.com/rancher/ui-plugin-operator/pkg/version.GitCommit={{ .ShortCommit }}"
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    binary: '{{ .ProjectName }}_{{ .Os }}_{{ if (eq .Arch "amd64_v1") }}amd64{{ else }}{{ .Arch }}{{ end }}'
+    no_unique_dist_dir: true
+
+archives:
+  - format: binary
+    name_template: '{{ .ProjectName }}_{{ .Os }}_{{ if (eq .Arch "amd64_v1") }}amd64{{ else }}{{ .Arch }}{{ end }}'
+
+checksum:
+  name_template: checksums.txt
+
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - "^docs:"
```

Comment:
Why are we not excluding tests?

Reply:
I assumed that `changelog.filters.exclude` was used to exclude commits that touch paths that match listed regex. However, the regex actually excludes commits based on commit messages, not paths (see https://goreleaser.com/customization/changelog/). Since we don't follow any convention for commit messages, this is doing nothing. I will remove it instead.